platformcomponents/win-hc/combobox.json

```diff
@@ -0,0 +1,20 @@
+{
+  "combobox": {
+    "figma": "https://www.figma.com/file/rNpQFybgpgSfTmhLa441V3/Components---Windows-OS-HighContrast?node-id=4931%3A2083",
+    "#hovered": {
```

Comment:
are we missing the #normal state here?

Reply:
Yes, it contains only the states that need an overwrite/a fix in HC(see the "Before" screenshot for ComboBox) - we do not overwrite all the states because it introduces a maintenance burden. Common states are inherited